_FIPS201/authentication.md

```diff
@@ -41,6 +41,9 @@ individuals and organizations that could occur as a result of errors in the auth
 cardholder. Once the required assurance level has been determined, one of the authentication mechanisms
 specified in [Section 6.2](authentication.md#s-6-2) **SHALL** be applied to achieve that assurance level.
 
+[^compensating]: As described in [Section 2.7](requirements.md#s-2-7), compensating controls **MAY** be applied for identity source documents evidence at IAL3 as the Standard requires only one piece of Strong evidence and one other piece of FAIR evidence. The issuance of PIV Card meets and exceeds IAL3; however,  since the federal background investigation serves as a strong compensating control for IAL3 ID proofing. 
```

Comment:
```suggestion
[^compensating]: As described in [Section 2.7](requirements.md#s-2-7), compensating controls can be applied for identity source documents evidence at IAL3 as the Standard requires only one piece of Strong evidence and one other piece of FAIR evidence. The issuance of PIV Card meets IAL3 since the federal background investigation serves as a strong compensating control for IAL3 ID proofing.
```

Reply:
Suggest also being consistent with capitalization of Strong and FAIR.